install_config/cluster_metrics.adoc

```diff
@@ -351,55 +351,54 @@ be added to your inventory file if it is necessary to override them.
 |Variable |Description
 
 |`openshift_metrics_install_metrics`
-|Deploy metrics if `true`, otherwise undeploy.
+|Deploy metrics if `true`. Otherwise, undeploy.
 
 |`openshift_metrics_start_cluster`
 |Start the metrics cluster after deploying the components.
 
 |`openshift_metrics_image_prefix`
-|The prefix for the component images. For
-example, with "openshift/origin-metrics-cassandra:v1.3", set prefix
-"openshift/origin-".
+|The prefix for the component images. For example, with
+"openshift/origin-metrics-cassandra:v1.3", set prefix `openshift/origin-`.
```

Comment:
Is this version of the documentation targeted at origin or enterprise?  Sorry, not familiar with how the doc source flow works.  If this is enterprise, we should re-do these examples.   It would be openshift3/openshift-metrics-cassandra:3.5.0.